pandas/tools/pivot.py

```diff
@@ -382,7 +382,9 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
     columns : array-like, Series, or list of arrays/Series
         Values to group by in the columns
     values : array-like, optional
-        Array of values to aggregate according to the factors
+        Array of values to aggregate according to the factors.
```

Comment:
@nickeubank this is too magical the way you have done it. Pls remove this automatic ``aggfunc='count'``. you can propose this in another issue if you want.